src/synthesizer/parametric/sf_hist.py

```diff
@@ -232,81 +240,50 @@ def _sfr(self, age):
         """
 
         # Set the SFR based on the duration.
-        if age <= self.duration:
+        if (age <= self.max_age) & (age > self.min_age):
             return 1.0
         return 0.0
 
 
 class Exponential(Common):
-    """
-    An exponential star formation history.
-
-    Attributes:
-        tau (float)
-            The "stretch" parameter of the exponential.
-    """
-
-    def __init__(self, tau):
-        """
-        Initialise the parent and this parametrisation of the SFH.
-
-        Args:
-            tau (unyt_quantity)
-                The "stretch" parameter of the exponential.
-        """
-
-        # Initialise the parent
-        Common.__init__(self, name="Exponential", tau=tau)
-
-        # Set the model parameters
-        self.tau = tau.to("yr").value
-
-    def _sfr(self, age):
-        """
-        Get the amount SFR weight in a single age bin.
-
-        Args:
-            age (float)
-                The age (in years) at which to evaluate the SFR.
-        """
-        return np.exp(-age / self.tau)
-
-
-class TruncatedExponential(Common):
     """
     A truncated exponential star formation history.
 
     Attributes:
         tau (unyt_quantity)
             The "stretch" parameter of the exponential.
         max_age (unyt_quantity)
-            The age at which the exponential is truncated. Above this age
-            the SFR is 0.
+            The age above which the star formation history is truncated.
+        min_age (unyt_quantity)
+            The age below which the star formation history is truncated.
     """
 
-    def __init__(self, tau, max_age):
+    def __init__(self, tau, max_age=1e11 * yr, min_age=0.0 * yr):
```

Comment:
Maybe the maximum age could be set to something reasonable as default

Reply:
Possibly, though in most cases it won't make a difference at all.

Reply:
Why are these being given defaults at all? At least for `max_age`. This is a important parameter of the model. It's totally reasonable you'd expect the user to always give it.

Reply:
I somewhat agree with `max_age`, and forcing its use makes people consider the age of the Universe in their particular scenario. changing `min_age` from 0 fundamentally changes the model though.

Reply:
Sure, `min_age` I get. I mainly mean `max_age`. It works better as a positional argument and means you don't need to muck about with defaults and/or `Nones`

Reply:
Done.